DigitalExcellence/dex-backend
# changed endpoint for getting al hightlights. Status code is now 200 w…

…hen the list is emtpy.

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Make sure there are no highlights in the database
2. Do a GET request to /api/Highlights (with right credentials)
3. Check if you receive an empty array with status code 200.

## Link to issue

Closes: #207
